autorest/codegen/serializers/__init__.py

```diff
@@ -72,6 +69,12 @@ def serialize(self, code_model: CodeModel) -> None:
             self._serialize_and_write_models_folder(code_model=code_model, env=env, namespace_path=namespace_path)
 
 
+
+    def _keep_patch_file(self, path_file: Path):
```

Comment:
the other file that needs to be updated is [this](https://github.com/Azure/autorest.python/blob/autorestv3/autorest/codegen/templates/init.py.jinja2#L11) one, so we read in the patch file in both sync and async. All you should have to do is remove the "if async_mode" and regenerate